scripts/environment.yml

```diff
@@ -23,6 +23,7 @@ dependencies:
   - r-stringr
   - r-purrr
   - pysam
+  - nodejs # build website locally
```

Comment:
`/site/README.md` already contains instructions on how to build the site locally. I don't believe just including `nodejs` here would be sufficient; you'd need to also install Bun for package management, and then run `bun install` to install the packages, and then run `bun run build`.

Reply:
Only nodejs is in conda, so it was getting deleted every time I updated the environment file while bun was remaining installed. So I added it here.

Reply:
I don't really know much about Conda, but I assume it wouldn't delete some random app on your computer? Node and Bun should be installed globally, as regular programs on your machine, not in the project folder or through an environment or something.

It doesn't matter to me that this line is here, but I'm just curious what's going on here.